app/src/org/odk/collect/android/widgets/IntentWidget.java

```diff
@@ -104,9 +98,6 @@ public void makeTextView() {
     public void makeButton(){
         setOrientation(LinearLayout.VERTICAL);
 
-        TableLayout.LayoutParams params = new TableLayout.LayoutParams();
-        params.setMargins(7, 5, 7, 5);
-
```

Comment:
What makes you think that? Could be totally true, but shouldn't take it out unless you're sure.